src/Controls/src/Core/PointerEventArgs.cs

```diff
@@ -11,15 +11,28 @@ public class PointerEventArgs : EventArgs
 
 		Func<IElement?, Point?>? _getPosition;
 
+		/// <summary>
+		/// Platform-specific arguments associated with the PointerEventArgs
+		/// </summary>
+		public PlatformPointerEventArgs? PlatformArgs { get; private set; }
+
+		/// <summary>
+		/// Public constructor that creates an instance of <see cref="PointerEventArgs"/>
```

Comment:
No need to state this is public since all visible APIs are public.  Constructors are pretty chill (especially those with no params), so you can just say 
```suggestion
		Initializes a new instance of the <see cref="PointerEventArgs"> class.
```
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor